htlc/src/components/withdrawHtlc.js

```diff
@@ -1,54 +1,56 @@
-/* global AlgoSigner */
-const { LEDGER } = require('../algosigner.config');
 const YAMLData = require('../../artifacts/cache/htlc.py.yaml');
-const algosdk = require('algosdk');
+const algosdk = require("algosdk");
+
+const token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+const server = "http://localhost";
+const port = 4001;
+let algodclient = new algosdk.Algodv2(token, server, port);
+
+const waitForConfirmation = async function (algodclient, txId) {
+  let response = await algodclient.status().do();
+  let lastround = response["last-round"];
+  while (true) {
+      const pendingInfo = await algodclient.pendingTransactionInformation(txId).do();
+      if (pendingInfo["confirmed-round"] !== null && pendingInfo["confirmed-round"] > 0) {
+          //Got the completed Transaction
+          console.log("Transaction " + txId + " confirmed in round " + pendingInfo["confirmed-round"]);
+          break;
+      }
+      lastround++;
+      await algodclient.statusAfterBlock(lastround).do();
+  }
+};
 
 async function WithdrawHtlc() {
   try {
-    const sndrAddr = prompt('Enter Escrow Address');
-    const recvAddr = prompt('Enter Receiver Address');
-    const secret = prompt('Enter Secret');
-    const secretBytes = new Uint8Array(Buffer.from(secret));
-
-    const txParams = await AlgoSigner.algod({
-      ledger: LEDGER,
-      path: `/v2/transactions/params`,
-    });
+    const sender =  "YEUJZXDQAUA3J6WK3RZQJKHCM3S7E6UQF3I6AKRGTIGACFXDKFSKBYCYKU";
+    const receiver = "EDXG4GGBEHFLNX6A7FGT3F6Z3TQGIU6WVVJNOXGYLVNTLWDOCEJJ35LWJY";
+    const secret = "hero wisdom green split loop element vote belt";
+    const secretBytes = new Uint8Array(Buffer.from(secret)); 
```

Comment:
Why are these values hardcoded?

Reply:
Well this is just an example of htlc, and the prompts were not doing good. I thought that when we will improve the UI, then we can change this. Should I change this?

Reply:
I think so, we can improve the UI later. But i think we should take input from the user (receiver, secret). It wouldn't be interactive otherwise. WDYT?

Reply:
Cool, I'll make the changes.